doc/nrf/known_issues.rst

```diff
@@ -3025,6 +3025,14 @@ Trusted Firmware-M (TF-M)
 
 .. rst-class:: v2-3-0
 
+NCSDK-18321: TF-M PSA architecture tests do not build with CMake v2.25.x
+  The :ref:`tfm_psa_test` fails to build with CMake version 2.25.x with missing header files.
+  This happens because the CMake install command is executed before the build command with the affected CMake versions.
+
+  **Workaround:** Use CMake version 2.26.x
```

Comment:
Sorry, I missed this somehow in the first round - add period at the end of the line for the workaround :)
Otherwise, LGTM.

Reply:
I'm also fine with approving once this is done.

Reply:
Done ;)